RecoTracker/MkFit/plugins/MkFitIterationConfigESProducer.cc

```diff
@@ -18,46 +18,38 @@ namespace {
                                         const TrackVec &in_seeds,
                                         const EventOfHits &eoh,
                                         IterationSeedPartition &part) {
-    // Seeds are placed into eta regions and sorted on region + eta.
-
     const size_t size = in_seeds.size();
 
     for (size_t i = 0; i < size; ++i) {
       const Track &S = in_seeds[i];
 
       const bool z_dir_pos = S.pz() > 0;
 
-      HitOnTrack hot = S.getLastHitOnTrack();
-      // MIMI ACHTUNG -- here we assume seed hits have already been remapped.
-      // This was true at that time :)
+      const auto &hot = S.getLastHitOnTrack();
       const float eta = eoh[hot.layer].GetHit(hot.index).eta();
-      // float  eta = S.momEta();
 
       // Region to be defined by propagation / intersection tests
       TrackerInfo::EtaRegion reg;
 
       // Max eta used for region sorting
       constexpr float maxEta_regSort = 7.0;
 
-      // Hardcoded for cms ... needs some lists of layers (hit/miss) for brl / ecp tests.
-      // MM: Check lambda functions/std::function
       const LayerInfo &outer_brl = trk_info.outer_barrel_layer();
 
-      const LayerInfo &tib1 = trk_info.m_layers[4];
-      const LayerInfo &tob1 = trk_info.m_layers[10];
+      // Define first (mkFit) layer IDs for each strip subdetector.
+      const int tib1_id = 4;
+      const int tob1_id = 10;
+      const int tecp1_id = 27;
+      const int tecn1_id = 54;
```

Comment:
Could be `constexpr`, but I'm not sure of the impact in generated code.

Reply:
Switched to constexpr, since it should make no difference.